e2e/protractor.base.conf.js

```diff
@@ -20,13 +20,11 @@ exports.config = {
     realtimeFailure: true
   },
   useAllAngular2AppRoots: true,
-  beforeLaunch: function () {
-    require('ts-node').register({
-      project: 'e2e/tsconfig.e2e.json'
-    });
-  },
   random: false,
   onPrepare() {
     jasmine.getEnv().addReporter(new SpecReporter({ spec: { displayStacktrace: true } }));
+    require('ts-node').register({
```

Comment:
Are these changes to the hooks needed for running tests in parallel?

Reply:
Yup. It doesn't work otherwise. This line of code must be prepared before each E2E Tests so that our imports don't fail. Previously this was done only once at the start of the `ng e2e` process but It causes the parallel tests to fail. Shifting it to onPrepare ensures that its carried out before each instance.

https://stackoverflow.com/questions/45336653/multicapabilities-configuration-error